pkg/drivers/common.go

```diff
@@ -264,6 +281,18 @@ func parseDHCPdLeasesFile(file io.Reader) ([]DHCPEntry, error) {
 	return dhcpEntries, scanner.Err()
 }
 
+func parseMACAddress(mac string) ([]byte, error) {
```

Comment:
Found https://pkg.go.dev/net#ParseMAC now, I'll replace it if we continue with this change.